lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -105,7 +110,8 @@ def build_container(machine_spec, machine_options)
         target_repository = 'chef'
         target_tag = machine_spec.name
 
-        image = find_image(target_repository, target_tag)
+        # check if target image exists, if not try to look up for source image.
```

Comment:
Were talking about it with @jkeiser at #27 thread. Seems he wasn't convinced. But I personally use this quite often. Otherwise there is no way to use local image as base image for other machines. 
I've updated example https://github.com/chef/chef-provisioning-docker/issues/27#issuecomment-109050291 , to make use case more clear.